src/common/config_opts.h

```diff
@@ -13,7 +13,7 @@
  */
 
 /* note: no header guard */
-OPTION(host, OPT_STR, "localhost")
+OPTION(host, OPT_STR, "default")  // "default" means that host will be `hostname -s` 
```

Comment:
hi, @liewegas thanks for your comments.  I'm agree with your advice that use $hostname and $fqdn.

but, there are a lot of $host config in user product env probably. Eliminating $host must be carefully.

in RGW, there is a option called `rgw dns name`, that is used in s3 authorization.In my dev/test env,  I would like to config this option to really hostname. so providing a $hostname is useful.